src/include/units/quantity.h

```diff
@@ -48,6 +48,158 @@ concept safe_divisible = // exposition only
     treat_as_floating_point<Rep> ||
     ratio_divide<typename UnitFrom::ratio, typename UnitTo::ratio>::is_integral();
 
+//  constrain operators to mpusz::units::quantity only
+template<typename T>
+concept ll_quantity_concept = detail::is_quantity<T>;
```

Comment:
This actually does not change anything as the `Quantity` concept is defined as:
https://github.com/mpusz/units/blob/5dd9eaac87a50adc62b170987ebac2d1344d63c5/src/include/units/concepts.h#L218-L232

so the definition is exactly the same. Actually, I thought that is your idea for the extension as everyone can partially specialize this type trait for his/her own quantity class and those functions will just start to work. Now I am a bit puzzled what was your intention with using a `Quantity` concepts instead of the `quantity` class template?